src/main/views/subscription-confirmation.njk

```diff
@@ -0,0 +1,53 @@
+{% extends "template.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "./macros/common-components.njk" import submitButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ govukBackLink({
+    text: text,
+    href: '#',
+    attributes: {
+      onClick: "history.back();event.preventDefault();"
+    }
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">{{ header }}</h1>
+  <h3 class="govuk-heading-s">{{ subheader }}</h3>
+<form  method="post">
+  <table class="govuk-table govuk-!-width-one">
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">{{ table.header1 }}</th>
+      <th scope="col" class="govuk-table__header">{{ table.header2 }}</th>
+      <th scope="col" class="govuk-table__header govuk-!-text-align-right">{{ table.header3 }}</th>
+      <th scope="col" class="govuk-table__header govuk-!-text-align-right">{{ table.header4 }}</th>
+    </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+
+      <tr class="govuk-table__row">
+        <td scope="row" class="govuk-table__cell--no-line">{{ searchResults.urn }}</td>
+        <td scope="row" class="govuk-table__cell--no-line">{{ searchResults.caseNumber }}</td>
+        <td class="govuk-table__cell--no-line govuk-!-text-align-right">{{ searchResults.caseName }}</td>
+        <td class="govuk-table__cell--no-line govuk-!-text-align-right"><a href="#">{{ removeText }}</a></td>
+      </tr>
+
+    </tbody>
+  </table>
+  {{ submitButton(buttonText) }}
```

Comment:
youre missing the functionality to add more subscriptions as per ticket https://tools.hmcts.net/jira/browse/PUB-737

Reply:
Nigel said we didn't have it yet

Reply:
the add button has been added

Reply:
add link added